test/t8_schemes/t8_gtest_input_equal_output.cxx

```diff
@@ -0,0 +1,96 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
```

Comment:
```suggestion
  Copyright (C) 2025 the developers
```

you can also add this in to your vscode snippets, making it easier to prevent using the wrong date.